src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/ExceptionDataContract.cs

```diff
@@ -872,5 +872,42 @@ private void Read()
                 throw new InvalidOperationException();
             }
         }
+
+        private string GetEscapedElementString(string stringToEscape)
+        {
+            var sb = new StringBuilder();
```

Comment:
#9125 was merged while I was typing the comment, so I'll repeat it here: Why not append directly to `_sb` and avoid the intermediate `StringBuilder` and `string` allocations?